server/models/Request.js

```diff
@@ -32,6 +32,11 @@ const requestSchema = new Schema({
     type: Number,
     required: true,
   },
+  created_date: {
+    type: Date,
+    default: Date.now,
+    required: true,
+  }
```

Comment:
you can replace all this by { timestamps: true }, mongodb will automatically add createdAt field to a document.
![Screenshot 2024-04-07 at 8 49 33 PM](https://github.com/hoangr1010/Cupid/assets/117782458/85c85aa6-cf85-4c82-984f-ee65f4b04bd4)

Reply:
Done

Reply:
Done